docker-compose.auth.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3.0'
+
+services:
+  grafana:
+    container_name: 'iotopen-datasource'
+    build:
+      context: ./.config
+      args:
+        grafana_image: ${GRAFANA_IMAGE:-grafana-enterprise}
+        grafana_version: ${GRAFANA_VERSION:-9.5.3}
+    ports:
+      - 3000:3000/tcp
+    environment:
+      GF_LOG_LEVEL: "debug"
```

Comment:
Defaulting to debug log-level should be avoided

Reply:
I think debug log level makes sense in a dev environment.

Reply:
Solved by adding example URLS instead